package.json

```diff
@@ -5,10 +5,9 @@
   "scripts": {
     "build": "webpack --config ./webpack.config.prod.js --progress --colors",
     "start": "webpack-dev-server --config ./webpack.config.dev.js & json-server --watch ./src/db.json --port 3006",
-    "startDev": "webpack-dev-server --config ./webpack.config.dev.js",
-    "startProd": "webpack-dev-server --config ./webpack.config.prod.js",
-    "startTemplate": "webpack-dev-server --config ./webpack.config.run.js",
-    "startTest": "node ./setup.js",
+    "startDev": "VERSION=Dev node ./setup.js",
```

Comment:
Running `npm run startDev` only starts the proxy server and not the webpack-dev-server. Doing `npm run start` seems to start the correct things and is functional in the local development environment. It's all quite confusing to be honest.

Reply:
I'll remove startDev, the start should be enough for the dev case.